wordpress-mobile/WordPress-iOS
# Removed white background from site icon image view

### Description

Addresses an inconsistency where in _My Sites_ (i.e., `BlogListViewController`), site icons do not have a background applied. On the Site Details (`BlogDetailsViewController`), however, a white background is applied to the image. This has an adverse impact on images with transparency.

It appears that the white background was introduced along with a border to make the placeholder site icon look nice. The fix applies the border when the placeholder is in use, and effectively removes it in other cases. In doing so, it improves consistency with both Calypso and the aforementioned in-app blog list.

To test:

- Checkout the branch and confirm that existing tests pass.
- Review the code and make sure everything looks okay.
- Manually test the app with a combination of sites that both include & omit site icons. Also test by adding & removing a site icon.

Update release notes:


### Screenshots

For context, the following shows the site list & details screens **before** the change was made.

![context](https://user-images.githubusercontent.com/221062/52925134-8eba0a00-32e4-11e9-85b6-cb8ee5adbf3a.png)

The following image depicts a site icon **before** & **after** the change is applied.

![beforeandafter](https://user-images.githubusercontent.com/221062/52925158-a72a2480-32e4-11e9-9e26-c15ac9a9ced7.png)
